syntax = "proto3";

service Object {
    rpc On(Empty) returns (RespondState) {}
    rpc Off(Empty) returns (RespondState) {}
    rpc SetAttribute(NewAttribute) returns (RespondAttribute) {}
}

message Empty {}

message RespondState {
    bool state = 1;
}

message NewAttribute {
    float value = 2;
}

message RespondAttribute {
    float value = 3;
}


/*
Executar na pasta protos para compilar
python -m grpc_tools.protoc -I. --python_out=../generated --grpc_python_out=../generated lamp.proto
*/
